docs/spec/index.md

```diff
@@ -7,6 +7,6 @@ keywords: registry, service, images, repository,  json
 # Docker Registry Reference
 
 * [HTTP API V2](api.md)
-* [Storage Driver](../storage-drivers/index.md)
+* [Storage Driver](https://docs.docker.com/registry/storage-drivers/index.md)
```

Comment:
Does this link get cleaned up later when publishing? This URL does not work as is, https://docs.docker.com/registry/storage-drivers/ does